signer/src/storage/in_memory.rs

```diff
@@ -241,7 +241,7 @@ impl super::DbRead for SharedStore {
             .lock()
             .await
             .deposit_request_to_signers
-            .get(&(txid.clone(), output_index))
+            .get(&(*txid, output_index))
```

Comment:
You _could_ implement `Deref` for `BitcoinTxId` and use deref coercion instead (i.e. implicit `*`) -- if you wanted.

Reply:
Hmmm, yeah I thought so too, but I don't think you can (I think I tried this before and I tried it again just now). Maybe the additional reference of the tuple confuses the compiler. I removed `Deref` since I did not need it but I'm not opposed to adding it back if the want/need arises.

Reply:
Ah weird, yeah there's something there that's confusing the compiler anyway